django/thunderstore/repository/models/cache.py

```diff
@@ -111,3 +117,184 @@ def drop_stale_cache(cls):
                 entry.delete()
         for entry in cls.objects.filter(community=None).iterator():
             entry.delete()
+
+
+class APIV1ChunkedPackageCache(SafeDeleteMixin):
+    community: Community = models.ForeignKey(
+        "community.Community",
+        related_name="chunked_package_list_cache",
+        on_delete=models.CASCADE,
+    )
+    index: DataBlob = models.ForeignKey(
+        "storage.DataBlob",
+        related_name="chunked_package_indexes",
+        on_delete=models.PROTECT,
+    )
+    chunks: DataBlobGroup = models.ForeignKey(
+        "storage.DataBlobGroup",
+        related_name="chunked_package_list_cache",
+        on_delete=models.PROTECT,
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        get_latest_by = "created_at"
+
+    CACHE_CUTOFF_HOURS = 3
+    UNCOMPRESSED_CHUNK_LIMIT = 14000000  # 14MB compresses into ~1MB files.
+
+    @classmethod
+    def get_latest_for_community(
+        cls,
+        community: Community,
+    ) -> Optional["APIV1ChunkedPackageCache"]:
+        try:
+            return cls.objects.filter(community=community.pk).latest()
+        except APIV1ChunkedPackageCache.DoesNotExist:
+            return None
```

Comment:
```python
return cls.objects.filter(community=community.pk).order_by("-created_at").first()
```
would've avoided the trouble of having to try/catch + model meta fields but this works too